test/protocol/FundsHandlerTest.js

```diff
@@ -480,11 +487,11 @@ describe("IBosonFundsHandler", function () {
         // Mock offer
         const { offer, offerDates, offerDurations, disputeResolverId, offerFees } = await mockOffer();
         offer.quantityAvailable = "2";
-
+        offer.id = "0";
```

Comment:
I prefer to have these explicitly so that when debugging I know that if I see an offer.id == "0" it means it was not set up, otherwise they'd have the value of id = "1" as set in the mock